test/integration/install-variants_test.js

```diff
@@ -29,7 +31,7 @@ suite('install-variants', function() {
           assert.fail(err);
           done(err);
         }
-        runCommand(binPath, ['install', '--variants', '--offline'], {cwd: tmpPath}).then(() => {
+        runCommand(binPath, ['install', '--variants'], {cwd: tmpPath}).then(() => {
```

Comment:
What was the symptom on Travis? This is indeed odd as the code path I setup was to try to avoid any environment specifics interfering with the tests.